### ISSUE TITLE
Validation errors: biblScope

### ISSUE THREAD

**Issue by emylonas** [Brown-University-Library/iip-texts#11]
- bshe0024
- bshe0025
- bshe0026
- bshe0027
- jeru0304
- unkn0106

 `<biblScope unit="page" type="insc">109</biblScope>`
should be 
 `<biblScope unit="page">109</biblScope>`
but I'm not sure if the correct value is "insc" or "page" 

Also the following seem to have no `@unit` value

- idum0099.xml
- idum0219.xml
- jaff0061.xml
- masa0645.xml
- masa0696.xml 
- mger0018.xml  
- mger0019.xml 
- mger0020.xml  
- mger0021.xml  
- mger0022.xml  
- mger0023.xml  
- mger0024.xml  
- mger0025.xml  
- mger0026.xml  
- mger0032.xml

**Comment by GaiaLembi** [comment]
All solved except bshe0024-0027, which I will need to check against the publication. She did used numbers in her dissertation, but I cannot figure out from the xml file whether who created it referred to page or number.

**Comment by emylonas** [comment]
Just to recap. These are not done yet. Waiting for Gaia to check against publication. When I redid the validation pass, I got a few more. There are 2 more bshe files, which I may not have copied properly last time. I also found 2 more that we didn't notice last time for some reason - 

bshe0021.xmlDescription: attribute "type" not allowed here; 
bshe0022.xmlDescription: attribute "type" not allowed here; 
bshe0024.xmlDescription: attribute "type" not allowed here;
bshe0025.xmlDescription: attribute "type" not allowed here; 
bshe0026.xmlDescription: attribute "type" not allowed here;
bshe0027.xmlDescription: attribute "type" not allowed here;

idum0099.xml
Description: value of attribute "unit" is invalid; 
Description: attribute "type" not allowed here; 

jaff0061.xml
Description: value of attribute "unit" is invalid; 
Description: attribute "type" not allowed here; 

masa0645.xml
Description: value of attribute "unit" is invalid; 
Description: attribute "type" not allowed here;
